Orange/widgets/data/owrank.py

```diff
@@ -417,13 +424,14 @@ def updateScores(self):
 
         self.ranksModel.wrap(model_array.tolist())
         self.ranksModel.setHorizontalHeaderLabels(('#',) + labels)
-        self.ranksView.setColumnWidth(0, 30)
+        self.ranksView.setColumnWidth(0, 40)
```

Comment:
@ajdapretnar Wide enough?

Reply:
Yes, but now it looks annoyingly wide. 😆 I'm such a brat. I'll merge this as it is a great solution! 👌